setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages, setup
+
+requirements = ["pycryptodome>=3.0.0", "eth-abi>=4.0.0"]
+
+pydantic_requirements = ["pydantic>=2.0.0"]
+
+tests_requirements = ["pytest-cov>=4.0.0", "pytest>=7.0.0"]
+
+all_requirements = pydantic_requirements
+
+
+VERSION = "0.1.0"
+DESCRIPTION = "MerkleTree implementation"
+LONG_DESCRIPTION = "Yet another package for MerkleTree, this time compatible with OpenZeppelin implementation"
+
+setup(
+    name="merkle-zeppelin",
+    version=VERSION,
+    description=DESCRIPTION,
+    long_description=LONG_DESCRIPTION,
+    author="akcelero",
+    author_email="akcelero@gmail.com",
+    license="MIT",
+    packages=find_packages(),
```

Comment:
If you want to make this package to be compatible with type checking, create a `py.typed` file and add it in the setup info:
```
    package_data={
        # PEP-0561: https://www.python.org/dev/peps/pep-0561/#packaging-type-information
        'package_name': ['py.typed'],
    },
```

Reply:
I will use it, thanks! : )